OSDOCS-17045-2: CQA 1 for MACH-4 Cluster API for Compute Machines (#114216)
openshift/openshift-docs: modules/machine-set-authoritative-api-machines.adoc

[id="machine-set-authoritative-api-machines_{context}"]
= Authoritative API types of compute machines

[role="_abstract"]

> The authoritative API of a compute machine is determined by the values of the `.spec.authoritativeAPI` and `.spec.template.spec.authoritativeAPI` fields in the Machine API compute machine set that creates it.

Reviewer: I agree with the verb change, but it adds passive voice. How about:
```suggestion
The values of the `.spec.authoritativeAPI` and `.spec.template.spec.authoritativeAPI` fields in a Machine API compute machine set determine the authoritative API of the compute machines that it creates.
```

Author: What is "it"? Removed "that it creates" for now.

Reviewer: The machine set - could do "that the machine set creates" to avoid the unclear referent. Good catch :D
